### PR TITLE
MiniApp-PWA Comparison table updated

### PR DESCRIPTION
**Pull request by espinr**
This PR is a review of the existing table, as I volunteered in #177 . Main changes:

- Status of W3C specifications updated.
- Some new MiniApp APIs included in the table
- Improved readability of the table with Yes/No/Partial values, including the details in a <summary><detail> structure inside the cells. Using homogeneous terms.
- Other minor editorial changes and enhancement of the code (adding `abbr`s, `hreflangs`, etc.)

### CODE REVIEW

**Comment by xfq** [comment]
Preview: https://htmlpreview.github.io/?https://raw.githubusercontent.com/espinr/miniapp/comparison-update/white-paper/comparison.html

File: white-paper/comparison.html [hunk 1]
```diff
@@ -15,14 +15,14 @@
       padding: 1em;
       align-items: center;
     }
-
+
```
Comment (xfq) [inline white-paper/comparison.html:18]: Looks like there are a lot of unrelated whitespace changes. Could you revert these changes to make this PR easier to review (and make `git blame` easier to view)?

Comment (espinr) [reply white-paper/comparison.html]: I´ve reverted that commit with the pretty format. Perhaps is not easy to review at a glance because I included some new structures in the code like 

```
<td>
   <detail>
      <summary>Yes/no to simplify the table</summary>
      The details with homogeneous descriptions (like examples of the API methods) and removing some texts that can cause confusion such as "(private implementation)".
   </detail>
</td>
``` 

Please, let me know if you want some other changes to simplify the review.

**Review by xfq — APPROVED** [review]
